### PR TITLE
Validator: Push BlockHeader on new signatures received

### PR DESCRIPTION
**Pull request by omerfirmak**
This probably should not be a validator only thing, but since it uses Nominator, it will have to do for now.

### CODE REVIEW

File: source/agora/consensus/protocol/Nominator.d [hunk 1]
```diff
@@ -690,27 +690,31 @@ extern(D):
         Params:
             block_sig = the structure with details of the block signature
 
+        Returns:
+            updated header
```
Comment (TrustHenry) [inline source/agora/consensus/protocol/Nominator.d]:
```suggestion
            Updated block header
```

**Review by TrustHenry — APPROVED** [review]
LGTM, Thanks

**Review by linked0 — APPROVED** [review]
LGTM

**Review by hewison-chris — COMMENTED** [review]
LGTM

**Pull request merged**